src/js/components/common/Header.js

```diff
@@ -76,6 +76,7 @@ class Header extends React.Component {
               {flagChecks.volunteerPageIsEnabled() && <NavLink path="/volunteer" name="Volunteers" />}
               {flagChecks.curriculumIsEnabled() && <NavLink path="/curriculum" name="Curriculum" />}
               {flagChecks.contactUsIsEnabled() && <NavLink path="/contact-us" name="Contact Us" />}
+              {flagChecks.contactUsIsEnabled() && <NavLink path="/blog" name="Blog" />}
```

Comment:
Great job for hiding this feature behind a flag!! ❤️

There are other parts to flag creation though. Check out [this guide](https://github.com/teencode-org/teencode/blob/ft-blog/tools/README.md) to adding feature flags on the Teencode frontend